edgelet/edgelet-http-workload/src/module/cert/server.rs

```diff
@@ -133,12 +131,25 @@ fn server_cert_extensions(
 
 #[cfg(test)]
 mod tests {
+    use crate::module::cert::CertificateResponse;
     use http_common::server::Route;
 
     use edgelet_test_utils::{test_route_err, test_route_ok};
 
     const TEST_PATH: &str = "/modules/testModule/genid/1/certificate/server";
 
+    const MODULENAME: &str = "testModule";
```

Comment:
Nit: should be MODULE_NAME

Also, replace all the other instances of "testModule" with this.

Reply:
done